LabBuilder/LabBuilder.psm1

```diff
@@ -798,8 +798,8 @@ function Download-LabResources {
     # Downloading Lab Resources
     Write-Verbose -Message $($LocalizedData.DownloadingLabResourcesMessage)
 
-    # Bootstrap Nuget
-    $null = Get-PackageProvider -Name NuGet -ForceBootstrap -Force
+    # Bootstrap Nuget # This needs to be a test, not a force 
+    # $null = Get-PackageProvider -Name NuGet -ForceBootstrap -Force
```

Comment:
Hmm. Then might need to work on a fix for this bit.  Were you running using an Admin PS Session?

Lets keep the line commented out for now as it works OK on my system and I can force the AppVeyor CI to install it another way.